.github/workflows/on-demand-pypi-source-push.yml

```diff
@@ -1,4 +1,4 @@
-name: Release
+name: PyPi images
```

Comment:
Just confirming, this'll not be used anymore after this PR merge right?

Reply:
Yes. But want to remove them in another PR.